src/acc/acc_stream.cpp

```diff
@@ -25,10 +30,8 @@ extern "C" int acc_stream_priority_range(int* least, int* greatest){
   *least = -1;
   *greatest = -1;
 
-#ifndef __HAS_NO_CUDA_STREAM_PRIORITIES
-  cudaError_t cErr = cudaDeviceGetStreamPriorityRange(least, greatest);
-  if (cuda_error_check(cErr)) return -1;
-  if (cuda_error_check(cudaGetLastError())) return -1;
+#ifndef __ACC_HAS_NO_STREAM_PRIORITIES
```

Comment:
CUDA 5.5 for which this flag was (according to CP2K's `INSTALL.md`) was released in 2013. I'd suggest to drop it and require CUDA >= 5.5, guessing that nobody actually still tests with a CUDA that old.

Reply:
Agreed, that makes a lot of sense.